srcpkgs/bottles/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'bottles'
+pkgname=bottles
+version=2.0.5
+revision=1
+wrksrc=Bottles-${version}
+build_style=meson
+hostmakedepends="gettext pkg-config glib-devel desktop-file-utils appstream-glib"
```

Comment:
I think `desktop-file-utils` and `appstream-glib` can be moved to `checkdepends`. They are only required to run tests from what I see [here](https://github.com/bottlesdevs/Bottles/blob/master/data/meson.build).

Reply:
Yeah they can be moved, the package still compiles fine.

The checking phase fails but that's a really minor problem with upstream related to the screenshots in the repository, so it's not a showstopper.